### PR TITLE
Initial events dashboard

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@hackclub/design-system": "^0.0.1-9",
     "axios": "^0.18.0",
-    "formik": "^0.11.11",
+    "formik": "1.0.0-alpha.3",
```
Comment (lachlanjc) [inline package.json]: Assuming that this doesn’t break any of our current forms?

File: src/components/admin/events/EventForm.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+const InnerForm = ({
+  values,
+  errors,
+  touched,
+  handleChange,
+  handleBlur,
+  handleSubmit,
+  isSubmitting
+}) => {
+  return (
```
Comment (lachlanjc) [inline src/components/admin/events/EventForm.js]: Tiny detail: can remove the return, use parens instead of braces

**Comment by lachlanjc** [comment]
I’m retrying the Netlify build, it’s their fault, not this branch’s

**Review by lachlanjc — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (maxwofford) [reply package.json]: Haven’t had issues when testing locally, but I think I’m going to roll back — I didn’t end up using the FastField components that this version adds, so it’s a negligible improvement

File: src/pages/admin/event.js [hunk 3]
```diff
@@ -61,7 +72,25 @@ export default class extends Component {
       case 'needsToAuth':
         return <Login userType="admin" />
       case 'success':
-        return <EventForm event={event} updateEvent={this.updateEvent} />
+        return (
+          <Fragment>
+            <ImageForm
+              name="logo"
+              height="60px !important"
```
Comment (lachlanjc) [inline src/pages/admin/event.js]: Native support for height coming in DSX!

File: src/pages/admin/event.js [hunk 4]
```diff
@@ -61,7 +72,25 @@ export default class extends Component {
...
+              width="60px"
```
Comment (lachlanjc) [inline src/pages/admin/event.js]: For pixel amounts (above the scale), use a number, like `={60}`

File: src/components/admin/events/ImageForm.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+import PropTypes from 'prop-types'
+import api from 'api'
+import storage from 'storage'
+
+const Label = Button.withComponent('label')
+
+const Box = B.extend`
+  height: ${props => props.height};
+  background-size: cover;
+  background-image: url(${props => props.imageurl});
```
Comment (lachlanjc) [inline src/components/admin/events/ImageForm.js]: Use the BackgroundImage component :)

**Comment by lachlanjc** [comment]
Great job as usual @MaxWofford!

**Pull request merged**